### PR TITLE
fix(docs): add description of manually running a pipeline to Managing…

### PR DESCRIPTION
**Pull request by avh121**
… Pipelines.

### CODE REVIEW

File: guides/user/pipeline/managing-pipelines/index.md [hunk 1]
```diff
@@ -165,3 +165,11 @@ pipeline.
 2. Click **Pipeline actions** in the upper right corner, and select
 **Rename**.
   ![](images/pipeline-actions.png)
+
+## Manually run a pipeline
+
+From the pipelines tab, click **Start Manual Execution** on the pipeline you
```
Comment (dorbin) [inline guides/user/pipeline/managing-pipelines/index.md]: s/pipelines/**Pipelines**/

**Review by dorbin — COMMENTED** [review]
Should this go, mabye, under /guides/user/pipeline/triggers/? That is, in the Triggers > Overview? I know this isn't triggering, exactly, but maybe just mention it in that overview? 

OR: change the name from "Triggering Pipelines" to "Running Pipelines," and you still have "Triggering...Pub/Sub" and "Triggering...Webhooks," but also "Running...Manually". 

Just a thought.

**Comment by avh121** [comment]
Discussed offline: resolved the question of where this info should go by keeping the how-to in the Managing Pipelines guide, but adding a link to it in the Pipeline Triggers Overview.

**Review by dorbin — APPROVED** [review]
👍

**Pull request merged**